### PR TITLE
 add dropDownDirection to DropDownPickerProps

### PR DESCRIPTION
**Pull request by denny351**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -192,7 +192,7 @@ class DropDownPicker extends React.Component {
 
         this.setState({
             isVisible: !this.state.isVisible,
-            direction: this.props.dropDownDirection || lowestPointOfDropdown < screenHeight ? 'top' : 'bottom',
+            direction: this.props.dropDownDirection ? this.props.dropDownDirection : lowestPointOfDropdown < screenHeight ? 'top' : 'bottom',
```
Comment (mikehardy) [inline src/index.js]: Thanks for posting this! Out of curiosity if the type in `index.d.ts` is "top" or "bottom" now (per 2nd commit) how is this ternary working? It doesn't seem like it has a boolean to work with anymore?

Comment (denny351) [reply src/index.js]: I've just updated this again. 
But here, `dropDownDirection` can be "top", "bottom" or undefined. And in this ternary, I am using the fact that "top" and "bottom" are truthy, and undefined is not.

Comment (mikehardy) [reply src/index.js]: Ah I see! I must admit, I forget javascript truthiness when I'm first scanning things  - that's a bit subtle but also does look correct

**Review by mikehardy — APPROVED** [review]
This looks good to me now - I've done one release here but it's still unfamiliar to me, please be patient if it takes a while to kick out a new version - patch-package (https://github.com/ds300/patch-package) may be your friend in the meanwhile - will be very safe since you're assured of a warning on a new version (via patch-package) and the code's already merged.

**Pull request merged**